kernel/packages/unity-interface/portableExperiencesUtils.ts

```diff
@@ -0,0 +1,136 @@
+import {
+  ContentMapping,
+  EnvironmentData,
+  LoadablePortableExperienceScene,
+  MappingsResponse,
+  SceneJsonData
+} from '../shared/types'
+import { getSceneNameFromJsonData } from '../shared/selectors'
+import { parseParcelPosition } from '../atomicHelpers/parcelScenePositions'
+import { UnityPortableExperienceScene } from './UnityParcelScene'
+import { forceStopParcelSceneWorker, getSceneWorkerBySceneID, loadParcelScene } from 'shared/world/parcelSceneManager'
+import { unityInterface } from './UnityInterface'
+import { resolveUrlFromUrn } from '@dcl/urn-resolver'
+
+declare var window: any
+window['spawnPortableExperienceScene'] = spawnPortableExperienceScene
```

Comment:
This is only until we have the feature fully built, right? If so, could we add a comment explaining just that?

Reply:
Yes!